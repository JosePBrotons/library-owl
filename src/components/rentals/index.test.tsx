import React from 'react';
import renderer from 'react-test-renderer';
import Rentals from '.';

test('renders book detail component correctly', () => {
    const rentalsCmp = renderer.create(<Rentals />).toJSON();
    expect(rentalsCmp).toMatchSnapshot();
});
