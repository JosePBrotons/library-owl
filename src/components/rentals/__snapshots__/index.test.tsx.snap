// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders book detail component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(235, 246, 251, 1)",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingHorizontal": 20,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/img/book.png",
      }
    }
    style={
      Object {
        "height": 150,
        "width": 150,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "rgba(69, 69, 69, 1)",
        "fontFamily": "Nunito-Bold",
        "fontSize": 16,
        "lineHeight": 22,
        "marginVertical": 20,
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    [missing "en.rentals.empty" translation]
  </Text>
</View>
`;
